src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -438,5 +439,63 @@ private void GroupDescriptionControls_SizeChanged(object sender, SizeChangedEven
         {
             SetTextHeight();
         }
+
+        /// <summary>
+        /// According to the current GroupStyle selected (or not selected) in the ContextMenu several actions can be executed.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void GroupStyleCheckmark_Click(object sender, RoutedEventArgs e)
+        {
+            var menuItemSelected = sender as MenuItem;
+            if (menuItemSelected == null) return;
+
+            var groupStyleItemSelected = menuItemSelected.DataContext as GroupStyleItem;
+            if (groupStyleItemSelected == null) return;
+
+            //Means that no GroupStyle (Default, Custom) has been selected the the clicked one will be selected
```

Comment:
`then the clicked one` typo :)

Reply:
@QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/12667/commits/5fc4b1e35ba1ad5b280d4b412fa73f762bed056e